composer.json

```diff
@@ -15,11 +15,11 @@
         "phly/http": "^0.12.0"
     },
     "require-dev": {
-        "cakephp/cakephp": "^2.5.7",
+        "cakephp/cakephp": "dev-network-http",
         "ext-curl": "*",
         "guzzle/guzzle": "^3.9.2",
-        "guzzlehttp/guzzle": ">=4.1.4,<6.0.0",
-        "guzzlehttp/streams": ">=1.5.0,<4.0.0",
+        "guzzlehttp/guzzle": ">=4.1.4,<6.0",
```

Comment:
@xabbuh AFAIR on https://getcomposer.org/doc/01-basic-usage.md#package-versions, this is not true. `4.1.4` will only allow `4.1.x` version and `^4.1.4` is similar to `>=4.1.4,<5.0`, so IMO, @GrahamCampbell is right here.